src/ToDoItem/ToDoItem.js

```diff
@@ -9,20 +9,43 @@ export const ToDoItem = (message) => {
   checkbox.type = "checkbox";
   checkbox.classList.add("to-do-element-checkbox");
 
+  const checkToDoElement = (e) => {
+    e.target.checked
+      ? (toDoMessage.style.textDecoration = "line-through")(
+          (toDoMessage.style.opacity = 0.4)
+        )
+      : (toDoMessage.style.textDecoration = "none");
+    toDoMessage.style.opacity = 1;
+  };
+
   // TODO: show added message from AddToDoField inside task
   const toDoMessage = document.createElement("p");
-  toDoMessage.innerText = "test message";
+  toDoMessage.classList.add("task-message");
+  toDoMessage.innerText = message;
+
+  //TODO: add button to edit TodoElement
+  const editButton = document.createElement("div");
+  editButton.classList.add("edit-button");
+
+  // const editToDoButton = (e) => {toDoMessage.contentEditable = e.target.checked ? false : true}
+
+  const editToDoButton = () => {
+    if (checkbox.checked == false) {
```

Comment:
if(checkbox.checked) { 
 return;
  }
  toDoMessage.contentEditable = true;
  toDoMessage.focus();

Reply:
done in this way